qa/suites/rados/downstream/downstream_qe/pg_merge.yaml

```diff
@@ -0,0 +1,34 @@
+roles:
```

Comment:
can you separate out `roles` and `tasks` in two separate yamls. put roles in cluster dir. and possibly make a create a different dir name than `downstream_qe`

Reply:
you can follow the patter from other suites dir structure.

Reply:
Created two folders 
     1.clusters 
      2. tasks
Added "2-node.yaml"  file in the cluster folder.File contain cluster node details required for test.

Reply:
Successfully executed the test case.
Log file - http://pulpito.ceph.redhat.com/skanta-2020-08-03_05:45:35-rados:downstream:downstream_qe-master-distro-basic-clara/

Reply:
Renamed the directory with "trier-2".